win32/win32.h

```diff
@@ -274,7 +274,7 @@ typedef struct
     {
         FILE  _public_file;
         char* _ptr;
-    };
+    } _u;
```

Comment:
This is not the commit to fix this, but it is undefined behavior for any symbol to begin with an underscore.  So, in theory all the similar symbols should be changed to like `u_`, which likely will never happen.

Reply:
"All identifiers that begin with an underscore are always reserved for use as identifiers
with file scope in both the ordinary and tag name spaces."

This means they're reserved a names at file scope (variables, functions, typedefs at file scope) and tag names, like `struct this`, `union that` or `enum andthis`.

They aren't reserved universally like names starting with underscore followed by an uppercase letter or starting with two underscores. (Or any name containing two underscores in a row in C++.)

But I've changed it for the new name anyway.